list.json

```diff
@@ -2,7 +2,7 @@
   // http://www.bloggingwv.com/big-list-of-disposable-temporary-email-services/
-  ["10minutemail.com", "disposeamail.com", "dontreg.com", "e4ward.com", "guerrillamail.com", "inbox2.info", "jetable.com", "kasmail.com", "killmail.net", "maileater.com", "mailexpire.com", "mailinator.com", "mailmoat.com", "mytrashmail.com", "netmails.net", "noclickemail.com", "nullbox.info", "pookmail.com", "shortmail.net", "sneakemail.com", "spambob.com", "spambob.org", "spambox.info", "spambox.org", "spambox.us", "spamex.com", "spamfree24.net", "spamfree24.org", "spamgourmet.com", "spamhole.com", "spammotel.com", "tempinbox.com", "temporaryforwarding.com", "temporaryinbox.com", "trashmail.net", "xemaps.com", "yopmail.com"],
+  ["10minutemail.com", "disposeamail.com", "dontreg.com", "e4ward.com", "guerrillamail.com", "inbox2.info", "jetable.com", "kasmail.com", "killmail.net", "maileater.com", "mailexpire.com", "mailinator.com", "mailmoat.com", "mytrashmail.com", "netmails.net", "noclickemail.com", "nullbox.info", "pookmail.com", "shortmail.net", "spambob.com", "spambob.org", "spambox.info", "spambox.org", "spambox.us", "spamex.com", "spamfree24.net", "spamfree24.org", "spamgourmet.com", "spamhole.com", "spammotel.com", "tempinbox.com", "temporaryforwarding.com", "temporaryinbox.com", "trashmail.net", "xemaps.com", "yopmail.com"],
   // http://www.warriorforum.com/main-internet-marketing-discussion-forum/147524-list-temporary-email-services-you-may-want-block-your-autoresponder-little-rant.html
   ["mailed.in", "mailinator.com", "trash2009.com", "mailexpire.com", "MailEater.com", "jetable.org", "onlatedotcom.info", "guerrillamailblock.com", "spamhole.com", "uggsrock.com", "TempEMail.net", "smapfree24.org", "smapfree24.de", "smapfree24.info", "smapfree24.com", "smapfree24.eu", "spamspot.com", "spam.la", "meltmail.com", "anonymbox.com", "dodgit.com", "E4ward.com", "GishPuppy.com", "haltospam.com", "mailnull.com", "MintEmail.com", "Sneakemail.com"],
```

Comment:
why does `Sneakemail.com` is still here?

Reply:
I found it two more times, removed in update.